requirements.txt

```diff
@@ -23,6 +22,42 @@ pytest-dependency==0.5.1
 python-multipart==0.0.5
 python-vlc==3.0.18121
 PyYAML==6.0
```

Comment:
Just a reminder this file has a lot of changes. Can you revert this file to the old one and see what is actually needed?

Reply:
You can do this with: `git checkout develop requirements.txt`